refm/api/src/_builtin/ARGF

```diff
@@ -848,10 +848,12 @@ ch を返します。
 
 @param arg 出力するオブジェクトを任意個指定します。
 
+#@since 2.3.0
+--- read_nonblock(maxlen, exception: true)              -> String | Symbol | nil
+--- read_nonblock(maxlen, outbuf, exception: true)      -> String | Symbol | nil
+#@else
 --- read_nonblock(maxlen)              -> String
 --- read_nonblock(maxlen, outbuf)      -> String
```

Comment:
IOの方ではこうまとめられてますね、合わせるのはどうでしょうか

https://github.com/rurema/doctree/blob/87fecbdf2df713746aee58cfd0e653ab6b20b709/refm/api/src/_builtin/IO#L1567-L1571

https://docs.ruby-lang.org/ja/latest/method/IO/i/read_nonblock.html

```suggestion
#@since 2.3.0
--- read_nonblock(maxlen, outbuf = nil, exception: true) -> String | Symbol | nil
#@else
--- read_nonblock(maxlen, outbuf = nil) -> String
```

Reply:
C実装だと引数を渡さなかった時を特別扱いしていることがあるので、大丈夫かなと思って確かめたら大丈夫そうでした。

```
% docker run -it --rm rubylang/all-ruby env ALL_RUBY_SINCE=ruby-2.0 ./all-ruby -e 'open("/dev/zero"){|f|p f.read_nonblock(1, nil)}'
ruby-2.0.0-p0       "\0"
ruby-2.0.0-p195     "\x00"
...
ruby-2.7.0          "\x00"
```